src/components/Login/GoogleLoginBtn.js

```diff
@@ -29,16 +24,19 @@ const GoogleBtn = ({
     const dateRegistered = new Date();
 
     if (response.accessToken) {
-      await loginUserAsync(email, password, {
-        userName,
-        givenName,
-        familyName,
-        postalCode,
-        dateRegistered,
-      });
-      if (!currentUser.isFailed && !currentUser.isFetching) {
-        closeModal();
-      }
+      await loginUserAsync(
+        email,
+        password,
+        {
+          userName,
+          givenName,
+          familyName,
+          postalCode,
+          dateRegistered,
+        },
+        modal.openedFrom,
+        itemDetail.ownerId
```

Comment:
I'm not really sure what this is for. Can you explain the use of itemDetail for getting the ownerId?

Reply:
I need it for checking that the person who logged in is not the owner of the posting:

![image](https://user-images.githubusercontent.com/39810669/87730541-f2b45c80-c77c-11ea-8983-c6f4a99e0a52.png)